dev/NumberBox/NumberBox_themeresources.xaml

```diff
@@ -58,6 +58,7 @@
     <x:Double x:Key="NumberBoxPopupHorizonalOffset">-21</x:Double>
     <x:Double x:Key="NumberBoxPopupVerticalOffset">-27</x:Double>
     <x:Double x:Key="NumberBoxPopupShadowDepth">16</x:Double>
+    <x:Double x:Key="NumberBoxMinWidth">120</x:Double>
```

Comment:
How did you choose this number? TextControlThemeMinWidth is 64

Reply:
I took the width of SpinButtons and Delete Buttons (40px) and multiplied it by 3. Perhaps we should get some design eyes on this.